modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -84,11 +86,55 @@
           with_okta_user(scopes) do |auth_header|
             expect(ClaimsApi::Veteran).to receive(:new).and_return(veteran)
 
-            get path, headers: auth_header
+            get all_claims_path, headers: auth_header
             expect(response.status).to eq(403)
           end
         end
       end
     end
+
+    context 'for a single claim' do
+      context 'when a known claimId is provided' do
+        it 'returns a 200' do
+          with_okta_user(scopes) do |auth_header|
+            expect(ClaimsApi::AutoEstablishedClaim)
+              .to receive(:get_by_id_or_evss_id).and_return({ id: '1111', claim_type: 'Appeals Control' })
+
+            get claim_by_id_path, headers: auth_header
+            expect(response.status).to eq(200)
+          end
+        end
+      end
+
+      context 'when a known BGS claimId is provided' do
+        it 'returns a 200' do
+          with_okta_user(scopes) do |auth_header|
+            expect(ClaimsApi::AutoEstablishedClaim).to receive(:get_by_id_or_evss_id).and_return(nil)
+            expect_any_instance_of(BGS::BenefitClaimWebServiceV1)
+              .to receive(:find_claim_details_by_claim_id).and_return({ bnft_claim_detail: {
+                                                                        bnft_claim_id: '1111',
+                                                                        bnft_claim_type_nm: 'Appeals Control'
+                                                                      } })
+
+            get claim_by_id_path, headers: auth_header
+            expect(response.status).to eq(200)
+          end
+        end
+      end
+
+      context 'when an unknown claim_id is provided' do
+        it 'returns a 404' do
+          with_okta_user(scopes) do |auth_header|
+            expect(ClaimsApi::AutoEstablishedClaim).to receive(:get_by_id_or_evss_id).and_return(nil)
+            expect_any_instance_of(BGS::BenefitClaimWebServiceV1)
+              .to receive(:find_claim_details_by_claim_id).and_return({})
+
+            get claim_by_id_path, headers: auth_header
+
+            expect(response.status).to eq(404)
+          end
+        end
+      end
+    end
```

Comment:
Should be 401 and 403 cases here as well i assume?

Reply:
I omitted them first pass since they technically are the same conditions as the other requests, didn't feel like they needed to be tested twice, but I don't mind adding them in

Reply:
added them in 👍